lite/examples/object_detection/android/README.md

```diff
@@ -37,5 +37,50 @@ Downloading, extraction and placing it in assets folder has been managed automat
 
 If you explicitly want to download the model, you can download from **[here](http://storage.googleapis.com/download.tensorflow.org/models/tflite/coco_ssd_mobilenet_v1_1.0_quant_2018_06_29.zip)**. Extract the zip to get the .tflite and label file.
 
+
+### Custom model used
+This example shows you how to perform TensorFlow Lite object detection using a custom model.
+* Clone the TensorFlow models GitHub repository to your computer.
+```
+git clone https://github.com/tensorflow/models/
+```
+* Build and install this repository.
+```
+cd models
+python3 setup.py build && python3 setup.py install
+```
+* Download the MobileNet SSD trained on **[Open Images v4](https://storage.googleapis.com/openimages/web/factsfigures_v4.html)** **[here](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md)**. Extract the pretrained TensorFlow model files.
+* Go to `models/research` directory and execute this code to get the frozen TensorFlow Lite graph.
+```
+python3 object_detection/export_tflite_ssd_graph.py \
+  --pipeline_config_path object_detection/samples/configs/ssd_mobilenet_v2_oid_v4.config \
+  --trained_checkpoint_prefix <directory with ssd_mobilenet_v2_oid_v4_2018_12_12>/model.ckpt \
+  --output_directory exported_model
+```
+* Convert the frozen graph to the TFLite model.
+```
+tflite_convert \
+  --input_shape=1,300,300,3 \
+  --input_arrays=normalized_input_image_tensor \
+  --output_arrays=TFLite_Detection_PostProcess,TFLite_Detection_PostProcess:1,TFLite_Detection_PostProcess:2,TFLite_Detection_PostProcess:3 \
+  --allow_custom_ops \
+  --graph_def_file=exported_model/tflite_graph.pb \
+  --output_file=<directory with the TensorFlow examples repository>/lite/examples/object_detection/android/app/src/main/assets/detect.tflite
+```
+`input_shape=1,300,300,3` because the pretrained model works only with that input shape.
+
+`allow_custom_ops` is necessary to allow TFLite_Detection_PostProcess operation.
+
+`input_arrays` and `output_arrays` can be drawn from the visualized graph of the example detection model.
+```
+bazel run //<directory with the TensorFlow repository>/lite/tools:visualize \
```

Comment:
Please add a note about how this helps visualization.

And its file to leave the path as `//tensorflow/` instead of `<directory with the TensorFlow repository>`, since we follow that convention everywhere else in the docs.

Reply:
```suggestion
bazel run //tensorflow/lite/tools:visualize \
```

Reply:
In detect.html we get the name of the input and output of the neural network (red rectangles).
![1](https://user-images.githubusercontent.com/10575983/74255790-2c87bf80-4d14-11ea-8a78-1a69e4ba16c3.png)
![2](https://user-images.githubusercontent.com/10575983/74255811-35789100-4d14-11ea-8112-120dae90aaff.png)